src/components/AutosuggestDeprecated/__mocks__/suggestions.js

```diff
@@ -11,16 +11,4 @@ export const SUGGESTIONS = [
     { name: 'Pluto', disabled: true },
 ];
```

Comment:
I think this file should move to some other location, rather then stay in a "deprecated" folder.